PyRoute/Calculation/RouteCalculation.py

```diff
@@ -150,6 +142,7 @@ def calc_trade(btn):
         return trade
 
     @staticmethod
+    @functools.cache
```

Comment:
Same here.

Reply:
Same reply - easy way to speed up without masses of code